src/south/FolderScanner/FolderScanner.class.js

```diff
@@ -34,6 +38,7 @@ class FolderScanner extends ProtocolHandler {
    */
   async onScan(scanMode) {
     this.logger.debug(`FolderScanner activated on scanMode: ${scanMode}.`)
+    this.lastOnScanAt = new Date().getTime()
```

Comment:
This variable can be maintained by the parent class ProtocolHandler (before calling onScan)